MAUI/ImageEditor/events.md

```diff
@@ -63,3 +63,27 @@ N> This is common for Shape and Text annotations.
 {% endhighlight %}
 
 {% endtabs %}
+
+## Browse image event
+
+This `BrowseImage` event occurs while browsing the image source when clicking on browse icon in toolbar.
+`Cancel`: We can restrict the default image browse picker opening by setting the `Cancel` argument `true`.
```

Comment:
`Cancel`: Restrict the default image browse picker opening by setting the `Cancel` argument `true.`

Reply:
updated.